BrCredentialCreatorWizard.vue

```diff
@@ -117,7 +119,70 @@ export default {
       return this.steps.length - 1;
     },
     steps() {
-      return [this.welcomeStep, ...this.flow, this.reviewStep];
+      const {welcomeStep, reviewStep} = this;
+
+      const steps = [];
+
+      // only add welcome step if available
+      if(welcomeStep !== null) {
+        steps.push(welcomeStep);
```

Comment:
This will push a `welcomeStep` with the value of `undefined`. Should this be `if(welcomeStep != null)`?

Reply:
@gannan08 -- `welcomeStep` is a computed property where `undefined` isn't an option (take a look at the computed property to see how a default step gets created).